openfecwebapp/templates/legal-advisory-opinions-landing.html

```diff
@@ -31,6 +31,14 @@
             <a class="u-float-right t-sans" href="{{url_for('advisory_opinions')}}">Advanced search</a>
           </div>
         </div>
+        <div class="row">
+          <div class="message message--info">
+            <h3>This feature is still in progress</h3>
+            <p>We are actively building the <span className="font-bold">advisory opinion search</span>, and it doesn&#39;t yet include some
```

Comment:
Should be `class` not `className` (thanks, React ;). Also, this could just be `<strong>` instead of `span` with a class.